src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -211,44 +211,45 @@ std::vector<std::optional<HashTable<HashedType>>> build(const RadixContainer<Lef
 
   for (size_t current_partition_id = 0; current_partition_id < radix_container.partition_offsets.size();
        ++current_partition_id) {
-    const auto partition_left_begin =
+    const auto build_partition_begin =
         current_partition_id == 0 ? 0 : radix_container.partition_offsets[current_partition_id - 1];
-    const auto partition_left_end = radix_container.partition_offsets[current_partition_id];  // make end non-inclusive
-    const auto partition_size = partition_left_end - partition_left_begin;
+    const auto build_partition_end = radix_container.partition_offsets[current_partition_id];  // make end non-inclusive
+    const auto build_partition_size = build_partition_end - build_partition_begin;
 
     // Skip empty partitions, so that we don't have too many empty hash tables
-    if (partition_size == 0) {
+    if (build_partition_size == 0) {
       continue;
     }
 
-    jobs.emplace_back(std::make_shared<JobTask>([&, partition_left_begin, partition_left_end, current_partition_id,
-                                                 partition_size]() {
-      auto& partition_left = static_cast<Partition<LeftType>&>(*radix_container.elements);
+    jobs.emplace_back(std::make_shared<JobTask>(
+        [&, build_partition_begin, build_partition_end, current_partition_id, build_partition_size]() {
+          auto& build_partition = static_cast<Partition<BuildColumnType>&>(*radix_container.elements);
```

Comment:
Is that indentation off?

Reply:
Na, it's okay. I wouldn't clang-format-off the entire Job-lambda just to get the lambda header right